src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedSimpleProgramEntryPointSymbol.cs

```diff
@@ -55,10 +55,14 @@ internal SynthesizedSimpleProgramEntryPointSymbol(SourceMemberContainerTypeSymbo
             bool isNullableAnalysisEnabled = IsNullableAnalysisEnabled(compilation, CompilationUnit);
             this.MakeFlags(
                 MethodKind.Ordinary,
+                RefKind.None,
                 DeclarationModifiers.Static | DeclarationModifiers.Private | (hasAwait ? DeclarationModifiers.Async : DeclarationModifiers.None),
                 returnsVoid: !hasAwait && !hasReturnWithExpression,
+                hasAnyBody: false,
```

Comment:
>hasAnyBody: false, [](http://example.com/codeflow?start=16&length=18)

If there was no body, we wouldn't have this symbol.

Reply:
'body' seems (in all other caseS) to be a bit marking if there is a BlockSyntax or ArrowSyntaxExpression originally available for a particular symbol.  I'm uncertain if that's the case here (esp. given that it always returned `false` for `IsExpressionBodied`).  So it seems weird to me that this symbol would use that bit in a subtly different way from teh rest of hte symbols.

Reply:
I have a feeling that `true` will have more chances to lead to correct behavior when the value will be used for this symbol.

Reply:
Done.